src/data/ground/ground_data_a01p01_station.h

```diff
@@ -0,0 +1,1488 @@
+
+
+
+
+
+
+
+
+static const struct ScriptCommand s_gs162_g0_s0_station_sref_script[] = { /* 0x820fd30 */
+    DEBUGINFO,
+    SELECT_MAP(162),
+    BGM_STOP,
+    JUMP_SCRIPT(COMMON_ENTER),
+};
+
+static const struct ScriptRef s_gs162_g0_s0_station_sref = { 404, 1, NULL /* ENTER_CONTROL */, s_gs162_g0_s0_station_sref_script }; /* 0x820fd9c */
+
+static const struct ScriptCommand s_gs162_g1_s0_station_sref_script[] = { /* 0x820fda8 */
+    DEBUGINFO,
+    SELECT_MAP(162),
+    SELECT_ENTITIES(-1, 0),
+    BGM_SWITCH(4),
+    { 0x22, 0x01,  0x003c,  0x00000000,  0x00000000, NULL },
+    { 0xe3, 0x00,  0x0003,  0x00000000,  0x00000000, NULL },
+    BGM_FADEOUT(60),
+    { 0x23, 0x01,  0x003c,  0x00000000,  0x00000000, NULL },
+    WAIT(90),
+    RET,
+};
+
+static const struct ScriptRef s_gs162_g1_s0_station_sref = { 400, 7, NULL /* EVENT_CONTROL */, s_gs162_g1_s0_station_sref_script }; /* 0x820fe48 */
+
+static const struct ScriptCommand s_gs162_g1_s0_lives0_dlg0[] = { /* 0x820fe54 */
+    DEBUGINFO,
+    { 0x53, 0x00,  0x0000,  0x01000000,  0x00000000, NULL },
+    { 0x54, 0x00,  0x0006,  0x00000000,  0x00000000, NULL },
+    { 0x2d, 0x07,  0x0000,  0x00000000,  0x00000000, NULL },
+    { 0xdf, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(60),
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("......") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("............") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("..................") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Where...") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Where am I?") },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Maybe...#W Is this a dream...?") },
```

Comment:
`#W` is mapped to the macro `WAIT_PRESS`. Can we use this macro in all these strings?